server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_4__add_odc_user_recent_browsing.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `odc_user_recent_browsing` (
+  `id` bigint NOT NULL AUTO_INCREMENT,
```

Comment:
always contains organization_id for table

Reply:
Okay，I'll make this change.